src/main/java/uk/ac/ox/oxfish/fisher/equipment/gear/factory/PurseSeineGearFactory.java

```diff
@@ -35,7 +33,6 @@
 public class PurseSeineGearFactory implements AlgorithmFactory<PurseSeineGear> {
 
     private Set<Observer<DeployFad>> fadDeploymentObservers = new HashSet<>();
```

Comment:
how do you make sure these stay clean between scenario runs?
What if people switch gear (don't dolphin setters do both?)

Reply:
> how do you make sure these stay clean between scenario runs?

Good point. Pesky mutable state. I think the `FadManager` will need to make copies.

> What if people switch gear (don't dolphin setters do both?)

Dolphin setters will just use `PurseSeineGear`, but I think the point is still valid.

Reply:
See if the Locker object can help. 
ITQMonoFactory uses it, inter alia, for more or less the same reason

Reply:
Resolved in acebbde